src/main/kotlin/domain/Board.kt

```diff
@@ -0,0 +1,50 @@
+package domain
+
+class Board(val height: Int, private val width: Int, private val mines: Mines) {
+
+    val rows = BoardRows(
+        List(height) { row ->
+            BoardRow(row, width, mines)
+        }
+    )
+
+    init {
+        fillNearMineCount()
+    }
+
+    private fun fillNearMineCount() {
+        repeat(height) { row ->
+            repeat(width) { col ->
+                calcNearCount(row, col)
+            }
+        }
+    }
+
+    private fun calcNearCount(row: Int, col: Int) {
+        for (direction in Directions.values()) {
+            val newRow = row + direction.row
+            val newCol = col + direction.col
+
+            increaseIfMine(
+                row,
+                col,
+                isInside(newRow, newCol) && rows[newRow][newCol] is BoardItem.Mine
```

Comment:
```suggestion
                isInside(newRow, newCol) && rows[newRow][newCol] is BoardItem.Mine
```
increaseIfMine 함수를 호출한다고 가정했을 때, isMine에 위 조건을 넣는 것이 잘 이해가 되지 않아요.
isMine을 판단하는 로직을 increaseIfMine으로 옮기면 어떨까요?